ci-operator/config/openshift/library-go/openshift-library-go-master.yaml

```diff
@@ -48,3 +48,10 @@ tests:
     make test-unit JUNITFILE=/tmp/artifacts/unit_report.xml --warn-undefined-variables
   container:
     from: test-bin
+- artifact_dir: /tmp/artifacts
+  as: e2e-aws-encryption
+  commands: |
+    mkdir -p /tmp/artifacts
+    make test-e2e-encryption JUNITFILE=/tmp/artifacts/unit_report.xml
```

Comment:
nit: `--warn-undefined-variables` helps to prevent bugs (as used in other targets)